### PR TITLE
Names of metrics in the new tracing system

### PR DESCRIPTION
**Pull request by jutaro**
This pull request aims to harmonize the names of metrics in the new tracing system with those in the old tracing system. Additionally, it addresses missing components in the metrics and fixes various errors. Key enhancements include the ability to add a prefix to metrics names from the configuration and the addition of type-specific suffixes to metrics names.
Changes

- Harmonization of Metric Names:
   Updated metric names in the new tracing system to align with the naming conventions used in the old tracing system.

- Completion of Missing Metrics:
  Added previously missing parts to the metrics to ensure comprehensive tracking and consistency.

- Error Fixes:
  Resolved various errors that were present in the metrics system, improving reliability and accuracy.

- Trace Dispatcher Enhancements:
  Introduced the ability to add a prefix to metric names based on configuration settings.
   Added type-specific suffixes (_int, _counter, _real) to metric names for better clarity and differentiation

### CODE REVIEW

File: cardano-node/src/Cardano/Node/Tracing/Tracers/ForgingThreadStats.hs [hunk 1]
```diff
@@ -73,13 +73,13 @@ instance MetaTrace ForgeThreadStats where
...
+      [("nodeCannotForge",
         "How many times was this node unable to forge [a block]?")
-      ,("Forge.NodeIsLeaderNum",
+      ,("nodeIsLeader",
         "How many times was this node slot leader?")
-      ,("Forge.BlocksForgedNum",
+      ,("blocksForged",
         "How many blocks did this node forge?")
-      ,("Forge.SlotsMissed",
+      ,("lotsMissed",
```
Comment (johnalotoski) [inline cardano-node/src/Cardano/Node/Tracing/Tracers/ForgingThreadStats.hs:82]: Typo, `slotsMissed`

Comment (jutaro) [reply cardano-node/src/Cardano/Node/Tracing/Tracers/ForgingThreadStats.hs]: Thanks for spotting this. Now corrected

File: trace-dispatcher/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Revision history for trace-dispatcher
 
+## 2.6.0
+
+* With a metrics prefix that can be set in the configuration (tcMetricsPrefix)
+  Metrics gets a type postfix (_int,_real, _counter)
```
Comment (mgmeier) [inline trace-dispatcher/CHANGELOG.md:6]: Just for context: these postfixes are alre part of the metrics names in legacy tracing?

File: trace-dispatcher/doc/trace-dispatcher.md [hunk 3]
```diff
@@ -263,6 +263,10 @@ class MetaTrace a where
 
 Metrics are seamlessly incorporated into the system through regular trace messages implementing the `asMetrics` function within the `LogFormatting` typeclass. Unlike other trace components, metrics are not subjected to filtering and are consistently provided. This occurs as long as the `EKGBackend` is configured for the message. The `EKGBackend` then forwards these metrics to `cardano-tracer` for additional processing. Subsequently, they are dispatched as Prometheus metrics, extending their utility and visibility.
 
+It is essential to implement the metricsDoc function of the MetaTrace typeclass, as this information is utilized to optimize system performance.
+
+The configuration option TraceOptionMetricsPrefix can be used to prepend a prefix to any trace message. For example, the prefix could be "cardano.node".
+
```
Comment (mgmeier) [inline trace-dispatcher/doc/trace-dispatcher.md:269]: Does the option prepend the prefix to any *trace message*, or just to any *metric name*?

**Review by mgmeier — APPROVED** [review]

File: trace-dispatcher/CHANGELOG.md (hunk 2 above)
Comment (jutaro) [reply trace-dispatcher/CHANGELOG.md:6]: Partially, if you go ekg_direct it is own responsibility

Comment (mgmeier) [reply trace-dispatcher/CHANGELOG.md:6]: Thanks!

**Pull request merged**